upstairs/src/backpressure.rs

```diff
@@ -67,19 +73,25 @@ pub struct BackpressureConfig {
 
 impl Default for BackpressureConfig {
     fn default() -> BackpressureConfig {
+        // `max_value` values below must be higher than `IO_OUTSTANDING_MAX_*`;
+        // otherwise, replaying jobs to a previously-Offline Downstairs could
+        // immediately kick us into the saturated regine, which would be
```

Comment:
what is "regine"

Reply:
Either "region" or "regime", take your pick (switched to "regime" in dc3130f1bcea)